vmm/Cargo.toml

```diff
@@ -28,6 +28,7 @@ micro_http = { git = "https://github.com/firecracker-microvm/micro-http", branch
 net_util = { path = "../net_util" }
 pci = {path = "../pci", optional = true}
 qcow = { path = "../qcow" }
+seccomp = { git = "https://github.com/firecracker-microvm/firecracker", branch = "master" }
```

Comment:
Is dependabot going to tell us this is out of date everyday?

Reply:
Yes until Firecracker folks move this to its own repo (which is already in their plan).

Reply:
Can we use a tag instead of "master" ?